estimagic/logging/database_utilities.py

```diff
@@ -223,6 +223,25 @@ def read_new_rows(
     fast_logging=False,
     limit=None,
 ):
+    """Read all iterations after last_retrieved up to a limit.
+
+    Args:
+        database (sqlalchemy.MetaData)
+        table_name (str): name of the table to retrieve.
+        last_retrieved (int): The last iteration that was retrieved.
+        return_type (str): either "list_of_dicts" or "dict_of_lists".
+        limit (int): Only the first ``limit`` rows will be retrieved. Default None.
```

Comment:
Limit is documented twice

Reply:
Addressed in 9f3c5cbb9b9c3fe40548ed30da551737dfee5523